etterlattemaler/src/main/kotlin/no/nav/pensjon/etterlatte/maler/tilbakekreving/TilbakekrevingVedlegg.kt

```diff
@@ -1,17 +1,23 @@
 package no.nav.pensjon.etterlatte.maler.tilbakekreving
 
+import no.nav.pensjon.brev.api.model.maler.Vedlegg
 import no.nav.pensjon.brev.template.Language.Nynorsk
 import no.nav.pensjon.brev.template.Language.English
 import no.nav.pensjon.brev.template.Language.Bokmal
+import no.nav.pensjon.brev.template.LanguageSupport
 import no.nav.pensjon.brev.template.createAttachment
+import no.nav.pensjon.brev.template.dsl.bokmal
+import no.nav.pensjon.brev.template.dsl.english
 import no.nav.pensjon.brev.template.dsl.helpers.TemplateModelHelpers
 import no.nav.pensjon.brev.template.dsl.newText
+import no.nav.pensjon.brev.template.dsl.nynorsk
 import no.nav.pensjon.brev.template.dsl.text
 import no.nav.pensjon.etterlatte.maler.tilbakekreving.TilbakekrevingDTOSelectors.perioder
 import no.nav.pensjon.etterlatte.maler.tilbakekreving.TilbakekrevingDTOSelectors.summer
+import kotlin.text.Typography.paragraph
 
 @TemplateModelHelpers
-val tilbakekrevingVedlegg = createAttachment(
+val tilbakekrevingVedlegg = createAttachment<LanguageSupport.Triple<Bokmal, Nynorsk, English>, TilbakekrevingDTO>(
```

Comment:
Fasinerende at intellij foreslår å fjerne eksplisitte type parametre her. Lurer på hva som skjer om man tilfeldigvis har to selectors med samme navn, men forskjellig mål-klasse.

Uansett, bruke typealias for språk? `LangBokmalNynorskEnglish`

Reply:
Ja, vart litt overraska over å finne fleire stadar vi har createAttachment utan typane definert. Introduserte det litt usystematisk, mest der det var enklaste måte å finne dto-en på, men liker det jo betre med.

Fint med typealias, skulle ønskje autocomplete lærte seg den.